specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/examples/GetFolder.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2019-04-01",
+    "subscriptionId": "{subscription-id}",
```

Comment:
The value should match the one in id, which is `b85c4986-56ae-4ebd-b5c5-a1595ca3dab1`